app/core/components/OnboardingQuests.jsx

```diff
@@ -587,13 +587,14 @@ const OnboardingSupporting = ({ data }) => {
             </div>
           </div>
           <div className="block text-right text-green-700 dark:text-green-200">
-            <div className="mt-3 text-sm md:mt-0 md:ml-6">
+            <div className="mt-3 text-sm md:ml-6 md:mt-0">
               <SupportingMemberSignupModal
                 button={
                   <span className="underline">
                     Subscribe<span aria-hidden="true">&rarr;</span>
                   </span>
                 }
+                sponsored={true}
```

Comment:
Just want to double-check: Is this `true` for the testing purpose? (Otherwise, I'm guessing everyone seeing this component will get free supporting membership, even though they are not sponsors.)

Reply:
That's an oopsie! 😅